docs/experiments-clueweb09b.md

```diff
@@ -5,11 +5,14 @@ Indexing:
 ```
 nohup sh target/appassembler/bin/IndexCollection -collection CW09Collection \
  -input /path/to/cw09/ClueWeb09_English_1/ -generator JsoupGenerator \
- -index lucene-index.cw09b.pos+docvectors -threads 32 -positions -docvectors -optimize \
+ -index lucene-index.cw09b.pos+docvectors -threads 32 -storePositions -storeDocvectors -optimize \
```

Comment:
I think the option name changes are fine.
But have you done the regression test on any of the collections?